### PR TITLE
update http to https

### PR DESCRIPTION
**Pull request by puneet-pdx**
- update `http` to `https` - all doc references and geocode service url
- update bintray references
- little code refactor in PlacesActivity
- update Release.md

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Release 1.0.8
+
+- Updates doc to avoid all use of `http:`
```
Comment (esreli) [inline RELEASE.md:3]:
```suggestion
- Updates project to favor the use of secured `https` rather than `http`
```

**Review by esreli — CHANGES_REQUESTED** [review]
The changes in source look good to me. I've made one suggestion to the RELEASE doc, I have one general request and have one general question.

Because this work can be encapsulated into a patch release, would you please update the app version to go along with the release, `1.0.8`?

Why does the project configure the `GEOCODE_URL` in two places? Is there a single configuration file that could house this static URL?

File: RELEASE.md (hunk 1 above)
Comment (puneet-pdx) [reply RELEASE.md:3]: updated

**Comment by puneet-pdx** [comment]
> The changes in source look good to me. I've made one suggestion to the RELEASE doc, I have one general request and have one general question.
> 
> Because this work can be encapsulated into a patch release, would you please update the app version to go along with the release, `1.0.8`?

Good catch. Updated App version to `1.0.8`

> Why does the project configure the `GEOCODE_URL` in two places? Is there a single configuration file that could house this static URL?

Generally the place for that is in Strings.xml file. But in order to get the values for string resources you would require `Context` . This project is following MVP pattern and it is not recommended for having `Context` in the `Presenter` .

This would required some refactoring of the App to be able to retrieve the String values in the `Presenters`. Maybe use a `repository` object ?

**Comment by esreli** [comment]
> Maybe use a `repository` object ?

I've never head of this technique, what is it? If the level of effort is larger than extra small, I think we should leave the project as-is.

**Review by esreli — APPROVED** [review]
These changes look good. Other than the last conversation [item](https://github.com/Esri/nearby-android/pull/99#issuecomment-728377385), this PR appears ready.

**Pull request merged**